### PR TITLE
hw-mgmt: attributes: Fix cpld_num for MSB78* systems Set cpld_num for MSB78* to "2" Small cleanups in source code

### PR DESCRIPTION
**Pull request by sholeksandr**
Signed-off-by: Oleksandr Shamray <oleksandrs@nvidia.com>

### CODE REVIEW

**Comment by sholeksandr** [comment]
RERUN

File: usr/usr/bin/hw-management-chassis-events.sh [hunk 1]
```diff
@@ -281,36 +281,6 @@ find_eeprom_name_on_remove()
 	fi
 }
 
-find_eeprom_name_on_remove()
```
Comment (vadimp-nvidia) [inline usr/usr/bin/hw-management-chassis-events.sh:284]: We have find_eeprom_name_on_remove() twice, right?

Comment (sholeksandr) [reply usr/usr/bin/hw-management-chassis-events.sh:284]: Yes, by some mistake we have it twice.

**Review by vadimp-nvidia — APPROVED** [review]

**Comment by sholeksandr** [comment]
RERUN

**Pull request merged**